### DISCUSSION TITLE
How to appends new options for Image Picker on select tag from JQuery o Javascript

### DISCUSSION THREAD

**Discussion by Emanuel108** [community/community#32862] [Other Feature Feedback, Questions, & Ideas]
Hi Guys, I'm using image picker plugin to select images on my webapp with .NET 6. When I try to get new options from Ajax and then pass them to the select it doesn't work. How Do I make this? Heeeeelp.

**Comment by Emanuel108** [comment]
The link for the plugin that I'm using is: https://github.com/rvera/image-picker

**Reply by strong-728** [reply]
You can add yourself like this code snip.
if ($('select').eq(0).is($(this))) {
    $('#reps').append('<img src="' + a + x + '" />');
} else if ($('select').eq(1).is($(this))) {
    $('#reps').append('<img src="' + a + y + '" />');
} else if ($('select').eq(2).is($(this))) {
    $('#reps').append('<img src="' + a + z + '" />');
}

**Reply by Emanuel108** [reply]
Do you have an example more complete please?

**Reply by strong-728** [reply]
yes. I have. I can support you directly with my best. Please ping me with you email.

**Reply by Emanuel108** [reply]
It's emacordovamontiel@hotmail.com

**Comment by jfullstackdev** [comment]
you can raise the same question at the Issues tab and hopefully they will reply, as they know their app best or their regular users

https://github.com/rvera/image-picker/issues
